docs/manual_api/cpp_tile_edsl.rst

```diff
@@ -0,0 +1,1070 @@
+C++ Tile eDSL
+#############
+
+The C++ Tile eDSL (Embedded Domain Specific Language) provides developers with a
+
```

Comment:
Remove all the extra newlines.

Reply:
the conversion tool added them. I think I got them all :)